_plays/08.md

```diff
@@ -4,10 +4,12 @@ title: Make it easy to use
 ---
 
 ### Checklist
-1.  Provide reference implementation.  Show how to use the API.
-2.  Write modifiable exampls.
-3.  Even create cliend-side wrappers with use, if you have the resources.  This brings the service even closer to the end-use.
+1.  Provide reference implementation.  
```

Comment:
well I like how we do docs at Mapbox :) 

https://www.mapbox.com/api-documentation/
https://www.mapbox.com/mapbox.js/example/v1.0.0/